### PR TITLE
Add chart for k8s cluster role access

### PR DESCRIPTION
**Pull request by sblausten**
https://app.shortcut.com/larder/story/13847/create-a-helm-chart-for-permissions-setup-for-roadie-k8s

### CODE REVIEW

File: roadie-kubernetes/templates/cluster-role.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+kind: ClusterRole
+metadata:
+  name: {{ .Values.clusterRole.name }}
+rules:
+  {{- toYaml .Values.clusterRole.rules.set1 | nindent 2 }}
+  {{- toYaml .Values.clusterRole.rules.set2 | nindent 2 }}
+  {{- toYaml .Values.clusterRole.rules.set3 | nindent 2 }}
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
```
Comment (punkle) [inline roadie-kubernetes/templates/cluster-role.yaml:12]: should this be in a different file.. not sure. just asking.

File: sample-service/README.md [hunk 2]
```diff
@@ -2,7 +2,7 @@ The sample service is a small HTTP API which is used for testing various integra
 
 This repo contains a simple Helm chart for deploying the sample service on Kubernetes.
 
-The code for the sample service can be found at [RoadiqHQ/sample-service](https://github.com/roadiehq/sample-service).
+The code for the sample service can be found at [RoadieHQ/sample-service](https://github.com/roadiehq/sample-service).
```
Comment (punkle) [inline sample-service/README.md:5]: Will we be adding the broker in the next commit?

I think we need the following for the broker part:

```
{{ if .Values.broker.enabled }}
---
apiVersion: v1
kind: ServiceAccount
metadata:
  name: broker-pod
---
apiVersion: rbac.authorization.k8s.io/v1
kind: ClusterRoleBinding
metadata:
  name: broker-pod-role-binding
roleRef:
  apiGroup: rbac.authorization.k8s.io
  kind: ClusterRole
  name: {{ .Values.clusterRole.name }}
subjects:
  - kind: ServiceAccount
    name: broker-pod
    namespace: default
---
apiVersion: apps/v1
kind: Deployment
metadata:
  name: broker-client
  labels:
    app: broker
spec:
  replicas: 1
  selector:
    matchLabels:
      app: broker
  template:
    metadata:
      labels:
        app: broker
    spec:
      serviceAccountName: broker-pod
      containers:
      - name: broker
        image: roadiehq/broker:kubernetes
        imagePullPolicy: Always
        env:
          - name: BROKER_TOKEN
            value: {{ .Values.brokerToken }}
          - name: BROKER_SERVER_URL
            value: https://{{ .Values.tenantName }}.broker.roadie.so
        ports:
        - containerPort: 8000
{{ end }}
```

File: roadie-kubernetes/README.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+
+## Roadie K8s read access role
+
+This chart sets up read access to your Kubernetes cluster from Roadie so that the Backstage Kubernetes plugin can read information from your cluster.
```
Comment (punkle) [inline roadie-kubernetes/README.md:4]: We need to take a note to expand on this before we close out the epic.

File: roadie-kubernetes/templates/cluster-role.yaml (hunk 1 above)
Comment (sblausten) [reply roadie-kubernetes/templates/cluster-role.yaml:12]: Doesn't matter - its related to the cluster role. If you like I can

File: sample-service/README.md (hunk 2 above)
Comment (sblausten) [reply sample-service/README.md:5]: Yes we discussed already - want to get this out and then look at broker and test it etc.. https://app.shortcut.com/larder/story/14399/add-broker-chart-for-k8s

File: roadie-kubernetes/README.md (hunk 3 above)
Comment (sblausten) [reply roadie-kubernetes/README.md:4]: Sure I'll add a separate ticket for the broker - https://app.shortcut.com/larder/story/14399/add-broker-chart-for-k8s

Comment (punkle) [reply roadie-kubernetes/README.md:4]: I mean I feel we need to add details about how to use the chart here in the readme.

**Review by punkle — APPROVED** [review]

**Pull request merged**